inference-engine/src/transformations/src/transformations/common_optimizations/dropout_with_random_uniform_replacer.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/dropout_with_random_uniform_replacer.hpp"
+
+#include <memory>
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/rt_info.hpp>
+
+#include "itt.hpp"
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::DropoutWithRandomUniformReplacer, "DropoutWithRandomUniformReplacer", 0);
+
```

Comment:
I do propose to add a comment here with a graph before and after the transformation. It really helps to clarify what dropout block we have and understand the code. Please take a look `convert_quantize_dequantize.cpp` as an example.

Reply:
we can copy-paste graphs along with descroption from removed DropoutWithRandomUniformReplacer.py and correct a bit. It will really simplify maintaince of this transformation and see that both graphs are functional equivalent. Now we have tendency not to do it but it is wrong.

Reply:
Added comment with transformation graph.